### DISCUSSION TITLE
The dynamic sync icon, iOS

### DISCUSSION THREAD

**Discussion by NWSS** [music-assistant/support#1067] [Feature requests and ideas]
At the moment, when MASS is syncing (which it does frequently) you get this dynamic circle on the top right. This is not a "calm" thing, it distracts. I would make this a static icon. For example: 

1. An open dot when nothing is happening, a full colored dot when there is some syncing or
2. No dot when nothing is happening, a full colored dot when there is some syncing..

Maybe the second one is my preference... but it's just an idea.

**Comment by OzGav** [comment]
relates to v1
